### PR TITLE
Hotfix transmit events

### PR DESCRIPTION
**Pull request by Baalmart**
# Hotfix transmit events

**_WHAT DOES THIS PR DO?_**
Just a few hotfixes for transmitting events from the device, mostly around error handling

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
[PLAT-1059]

**_HOW DO I TEST OUT THIS PR?_**
README

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [Bulk transmit measurements](https://app.gitbook.com/o/-MCogOVGQqjUharJKchL/s/-MKqyQkcfs54GYi-q96G/device-registry/events#transmit-values-in-bulk)
- [x] [Transmit single measurement](https://app.gitbook.com/o/-MCogOVGQqjUharJKchL/s/-MKqyQkcfs54GYi-q96G/device-registry/events#transmit-single-event)

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

N/A

**_ARE THERE ANY RELATED PRs?_**

N/A

[PLAT-1059]: https://airqoteam.atlassian.net/browse/PLAT-1059?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/device-registry/utils/create-event.js [hunk 1]
```diff
@@ -522,28 +525,41 @@ const createEvent = {
...
-
           return {
             success: false,
-            message: "Internal Server Error",
+            message: "Server Error",
             errors: {
-              message: errorMessage,
+              message: error.response
+                ? error.response.data.error.details
+                : "No active internet connection",
```
Comment (NoahNsimbe) [inline src/device-registry/utils/create-event.js:554]: `No active internet connection` might be missing leading to the API user.  If a user is able to get a response from the API it means there was in internet connection that enabled deliver of the response which contradicts with the API response message

Comment (Baalmart) [reply src/device-registry/utils/create-event.js]: Done. @NoahNsimbe

**Review by NoahNsimbe — APPROVED** [review]
Changes LGTM

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @Baalmart 
Looks good to me

**Pull request merged**